frontend/src/components/ImagesComponent.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { FunctionComponent, ReactElement, useState } from "react";
+import styled from "styled-components";
+import { SEA_FOAM } from "../constants/colors";
+
+
+interface ImagesProp {
+  images: string[],
+}
+
+interface ImageProp {
+  highlight: boolean,
+}
+
+const Image = styled.img`
+  border-radius: 20px;
+  width: 100%;
+  height: 100%;
+  transform: scale(0.8);
+
+  :hover {
+    cursor: pointer;
+    opacity: 0.8;
+  }
+
+  ${({highlight}: ImageProp) => highlight && `
+    border: 10px solid ${SEA_FOAM};
+  `};
+`;
+
+const ImagesComponent: FunctionComponent<ImagesProp> = ({ images })  : ReactElement => {
+  let [selected, setSelected] = useState(-1);
```

Comment:
I think this selected and setSelected should be one level above since there is no way that the parent component can see this state, which will be important later.

Reply:
It needs to know which was selected when sending this to firebase to know if it was right. Unless I am missing something.

Reply:
so ya i thought about it. To know which one is selected, it's just the selected variable right. To update the corresponding word, we can pass in the word ID a level up so that this ImagesComponent can have the word ID and make calls here